public/index-main.html

```diff
@@ -7,6 +7,13 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#241a5f" />
     <meta name="description" content="Oiretutka-palvelu auttaa ymmärtämään miten paljon Suomessa sairastetaan" />
+    <meta
+      property="og:title"
+      content="Oiretutka näyttää missä Suomessa sairastetaan - Auta seurantaa vastaamalla kyselyyn"
+    />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://www.oiretutka.fi/" />
+    <meta property="og:image" content="https://www.oiretutka.fi/static/media/oiretutka-logo-gradient.22532776.svg" />
```

Comment:
This should be based on `%PUBLIC_URL%`; the old asset is not guaranteed to continue to exist :thinking: